test/probes/pino.test.js

```diff
@@ -117,51 +90,25 @@ describe(`pino v${version}`, function () {
   })
 
   before(function () {
-    //
-    // make decisions based on pino version
-    //
+    // make the logger
+    logger = pino()
+
+    // modify the logger so that it emits logging so it can be checked. implement
+    // only the functions that are called.
+    const modStream = {
+      write (s) {
+        logEmitter.emit('test-log', s)
+      },
+      flush () {},
+      flushSync () {}
+    }
+
     if (major >= 5) {
-      // make the logger
-      logger = pino()
-
-      //
-      // modify the logger so that it emits logging so it can be checked. implement
-      // only the functions that are called.
-      //
-      stream = logger[streamSym] = {
-        write (s) {
-          logEmitter.emit('test-log', s)
-          if (debugging) {
-            console.log(s);  // eslint-disable-line
-          }
-        },
-        flush () {},
-        flushSync () {}
-      }
-      Object.setPrototypeOf(stream, EventEmitter.prototype)
+      logger[pino.symbols.streamSym] = modStream
+      Object.setPrototypeOf(logger[pino.symbols.streamSym], EventEmitter.prototype)
     } else if (major >= 2) {
```

Comment:
still needed? isn't it all >= 5 now?

Reply:
Thought about removing it - but decided to keep it as it works. We'll need to make a decision regarding "floor" of versions across all probes and clean there. Someday.

Reply:
Versions before 5 are still instrumented.

I thought about removing them - but since they work - I'll keep it for now.
We need to make a decision regarding "floor" versions across all probes and how to handle it. Someday.